comms/discovery/db/queries/get_follows.go

```diff
@@ -20,3 +21,47 @@ func CountFollows(q db.Queryable, ctx context.Context, arg CountFollowsParams) (
 	err := q.GetContext(ctx, &count, countFollows, arg.FollowerUserID, arg.FolloweeUserID)
 	return count, err
 }
+
+const bulkGetFollowers = `
+select
+  follower_user_id,
+	followee_user_id,
+	count(*)
+from follows
+where 
+	followee_user_id = :FolloweeUserID
+  and follower_user_id in (:FollowerUserIDs)
+	and is_current
+	and not is_delete
+group by follower_user_id, followee_user_id;
```

Comment:
I think this group by and count is not strictly necessary... but after looking at the code further down I think this all can work as is so seems fine.